test/src/sdk-harness/test_projects/tx_fields/mod.rs

```diff
@@ -251,7 +254,7 @@ async fn can_get_tx_output_type() {
     // Contract output
     let output_type = 1;
     let result_ptr = contract_instance
-        .get_tx_output_pointer(1)
+        .get_tx_output_pointer(0)
```

Comment:
@digorithm is this change expected? Are the outputs added in a different order now?

Reply:
We did touch on some parts related to that in recent changes, but I'm not sure about the context here -- what's this test doing? What's the expected behavior?

Reply:
This methods basically calls the following standard library function: https://github.com/FuelLabs/sway/blob/master/sway-lib-std/src/tx.sw#L277-L283.
It just looks like the tx output at index 0 is now of "type 1" and the tx output at index 1 is of "type 3". Previously, they were in the opposite order, that's all.